src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    int getArea();

```

Comment:
Cool, method name became much more meaningful, however, what do you think about further enhancing it to
`int calculateArea();`
It could be more clear, that area itself is not being stored somewhere, but rather is calculated each time method is called.

Reply:
@flekros Method names are typically verbs or verb phrases. For example, `sendMessage` or `stop`

Reply:
Okay,i got you.